### DISCUSSION TITLE
New motion events w/ Scrypted?

### DISCUSSION THREAD

**Discussion by sageDieu** [mrlt8/docker-wyze-bridge#1020] [Q&A] [answered]
Hi! I've been using Wyze bridge with Scrypted to get my wyze cams into homekit with great success for months (amazing project, thank you!!)

I saw the new release includes support for motion events from the API, and I was wondering if that's something I should care about as a Scrypted user - I've been playing around and couldn't find any obvious way to configure it in Scrypted. 

I was thinking it would be nice if this worked properly so that I can reduce the overhead of prebuffer+running my own motion detection on my server with Scrypted plugins, with an assumption that the more built-in option would be more performant for the cameras. If that isn't true then no reason to bother here.

**Comment by mrlt8** [comment, marked as answer]
`MOTION_API` has some limitations (most noticeably the 300 second cooldown for non-CamPlus cameras) which will probably make motion detection in scrypted/frigate better for most situations. 

However, it may be useful for battery-based cams since it will allow you to get motion events without draining the battery or if your network is struggling to handle streaming from multiple cameras or you're trying to monitor double digit cameras. 

I haven't really looked into the scrypted's MQTT, but you should be able to get the motion events into scrypted using the Webhook Plugin. 

btw, enabling pre-buffer in scrypted should also have the added benefit of making the camera feed load faster in the home app, so you may want to keep that enabled either way.

**Reply by sageDieu** [reply]
Thanks for responding, this makes sense! In my case I only have 5 cameras and none are battery based, and everything works fine with prebuffer, substreams, and openCV motion detection. If there isn't performance upside that you've noticed then it's probably not worth switching setups.
